File: CiderAndCode.Web/app/controllers/ViewBushels/viewBushelsController.js
app.controller("viewBushelsController",["$http","$scope",function($http, $scope) {
$scope.bushel = {};
$scope.users = [];

$http.get("/api/viewBushels")
    .then(function (result) {
        $scope.bushels = result.data;
        console.log("result.data", result.data)
    })

/*
$scope.pickApples = function () {
    let bushel = $scope.bushel;
    $http.post("/api/PickApples",
        {
            Type: bushel.appleType,
            NumberOfBushels: bushel.numberOfBushels,
            UserId: bushel.userId
        })
        .then(result => console.log(result))
        .catch(error => console.log(error));
};
    */

}]);

File: CiderAndCode.Web/Controllers/ViewBushelsController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net;
using System.Net.Http;
using System.Web.Http;
using CiderAndCode.Web.DataModels;
using CiderAndCode.Web.ViewModels;
using System.Data.Entity;

namespace CiderAndCode.Web.Controllers
{
    [RoutePrefix("api/viewBushels")]
    public class ViewBushelsController : ApiController
    {
        [HttpGet, Route("")]
        public HttpResponseMessage GetAllBushels()
        {
            var db = new AppDbContext();

            var bushels = db.Bushels.Select(bushel => new AppleResult
            {
                Id = bushel.Id, 
                TypeOfApple = bushel.Type.ToString(),
                NumberOfBushels = bushel.Quantity,
                ContributingUser = bushel.User.Name
            });

            return Request.CreateResponse(HttpStatusCode.OK, bushels);

        }



    }
}

    
